File: demo/src/main/java/com/otaliastudios/cameraview/demo/MessageView.java
package com.otaliastudios.cameraview.demo;


import android.content.Context;
import androidx.annotation.Nullable;
import android.util.AttributeSet;
import android.view.ViewGroup;
import android.widget.LinearLayout;
import android.widget.TextView;

public class MessageView extends LinearLayout {

    private TextView message;
    private TextView title;

    public MessageView(Context context) {
        this(context, null);
    }

    public MessageView(Context context, @Nullable AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public MessageView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        setOrientation(VERTICAL);
        inflate(context, R.layout.option_view, this);
        ViewGroup content = findViewById(R.id.content);
        inflate(context, R.layout.spinner_text, content);
        title = findViewById(R.id.title);
        message = (TextView) content.getChildAt(0);
    }

    public void setTitleAndMessage(String title, String message) {
        setTitle(title);
        setMessage(message);
    }

    public void setTitle(String title) {
        this.title.setText(title);
    }

    public void setMessage(String message) {
        this.message.setText(message);
    }
}

File: demo/src/main/kotlin/com/otaliastudios/cameraview/demo/MessageView.kt
package com.otaliastudios.cameraview.demo

import android.content.Context
import android.util.AttributeSet
import android.view.View
import android.view.ViewGroup
import android.widget.LinearLayout
import android.widget.TextView

class MessageView @JvmOverloads constructor(
        context: Context,
        attrs: AttributeSet? = null,
        defStyleAttr: Int = 0
) : LinearLayout(context, attrs, defStyleAttr) {

    init {
        orientation = VERTICAL
        View.inflate(context, R.layout.option_view, this)
        val content = findViewById<ViewGroup>(R.id.content)
        View.inflate(context, R.layout.spinner_text, content)
    }

    private val message: TextView = findViewById<ViewGroup>(R.id.content).getChildAt(0) as TextView
    private val title: TextView = findViewById(R.id.title)

    fun setMessage(message: String) { this.message.text = message }

    fun setTitle(title: String) { this.title.text = title }

    fun setTitleAndMessage(title: String, message: String) {
        setTitle(title)
        setMessage(message)
    }
}
